tasmota/tasmota_xdrv_driver/xdrv_09_timers.ino

```diff
@@ -888,11 +889,7 @@ void HandleTimerConfiguration(void)
 #else
   WSContentSend_P(HTTP_FORM_TIMER3);
 #endif  // USE_SUNRISE
-#ifdef USE_UNISHOX_COMPRESSION
```

Comment:
Why is this removed? The compressed code is smaller than the original so that's why it is here.

Reply:
Oh I fear you compressed the current HTTP_FORM_TIMER4[]. In that case the language specific D_HOUR_MINUTE_SEPARATOR is hardcoded in the compressed code and is wrong for other languages. Pls revert.

Reply:
Oh yes, you're right. I now use the same compressed and uncompressed code of HTTP_FORM_TIMER4. This makes the  code simpler and should avoid such mistakes in future.